lms/templates/courseware/course_about.html

```diff
@@ -105,6 +105,8 @@
 <%block name="pagetitle">${course.display_name_with_default_escaped}</%block>
 
 <section class="course-info">
+  <%block name="course_info_before_header"></%block>
```

Comment:
Why do we need an empty block right before the next block? Can we not include the content we want to add to `course_info_before_header` to the top of the `header` block?

Reply:
Ok, I changed it as you said.